gluten-core/src/main/scala/io/glutenproject/expression/ConverterUtils.scala

```diff
@@ -316,7 +316,10 @@ object ConverterUtils extends Logging {
         case BinaryType =>
           typedFuncName.concat("vbin")
         case DecimalType() =>
-          typedFuncName.concat("dec")
+          val decimalType = datatype.asInstanceOf[DecimalType]
+          val precision = decimalType.precision
+          val scale = decimalType.scale
+          typedFuncName.concat("dec<" + precision + "," + scale + ">")
```

Comment:
It seems like needs to support this name for ch backend. @taiyang-li

Reply:
ok, let's do it